docusaurus/docs/user-docs/releases/creating-a-release.md

```diff
@@ -3,12 +3,13 @@ title: Creating a release
 description: Instructions to create a release from the admin panel
 tags:
 - admin panel
+- Growth feature
```

Comment:
```suggestion
- Growth plan
```

I think we can update all tags with "Growth plan" and "Enterprise plan". Not necessary for this PR, I can do it repo-wide once this PR is merged 😊